.github/workflows/test.yml

```diff
@@ -63,3 +64,9 @@ jobs:
 
       - name: Run tests
         run: mix test --warnings-as-errors
+
+      - name: Cache Bumblebee models
+        uses: actions/cache@v4
+        with:
+          path: ${{ env.BUMBLEBEE_CACHE_DIR }}
+          key: ${{ runner.os }}-bumblebee
```

Comment:
This step restores the cache (then it is saved again in a post-step), so this should go before `mix test` so that we actually use the cache :D

Reply:
:facepalm: You are right...